example/README.md

```diff
@@ -0,0 +1,70 @@
+In this folder, we can see four main packages in different sub-folders:
+- register_certificate
+- register_contract
+- execute_contract
+- validate_ledger
+
+They demonstrate how to use Scalar DL Go Client SDK to send basic operation requests to Scalar DL networks.
+
+To build all of them
+```
+cd <sub-folder>
+go build
+```
+
+## Local Scalar DL Network
+We need a local Auditor-enabled Scalar DL network before we run these examples.
```

Comment:
Why do we need Auditor?

Reply:
I was thinking of demonstrating with an auditor-enabled Scalar DL network.
But, maybe we should make things simpler for the example.
I removed this part in https://github.com/scalar-labs/scalardl-go-client-sdk/pull/20/commits/cb68a0517e4133cb80e6df397edf91f3756ff310